packages/domain/lib/usecases/palindrome_usecase_impl.dart

```diff
@@ -1,13 +1,13 @@
 import 'package:domain/usecases/models/palindrome_usecase.dart';
 
 class PalindromeUsecaseImpl implements PalindromeUsecase {
```

Comment:
now it's ok, but in jenkins project try to use UseCase/UseCaseParams with generics as abstraction instead of PalindromeUsecase and override method call